### PR TITLE
Add unsafe C script

### PR DESCRIPTION
**Pull request by RagnarokMew**
Will add a unsafe script written in C. 

I should mention it doesn't work like I wanted it to btw maintainer ;-;.

### CODE REVIEW

File: scripts/script.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include<stdlib.h>
+
+int main(void)
+{
+    system("echo /usr/bin/ls");
```
Comment (robertmercea) [inline scripts/script.c:5]: The code is unsafe. Fix it.

**Review by robertmercea — CHANGES_REQUESTED** [review]
Fix the unsafe code pls

File: scripts/script.c (hunk 1 above)
Comment (RagnarokMew) [reply scripts/script.c:5]: Please specify which line of code is actually unsafe.
The intended unsafe code is the system() function. 
A self-respecting maintainer should be more precise in such matter. In the future please try to keep a standard of precision in your further endeavors lest  you be removed from the maintainers list of the repo

Comment (robertmercea) [reply scripts/script.c:5]: I see your point, but, unfortunately, your mother.

**Review by robertmercea — APPROVED** [review]
I win via "your mother" argument <3

**Pull request merged**